tests/userTest.js

```diff
@@ -575,7 +575,7 @@ describe('Main: Currently testing user management,', function () {
     await getCurrentUserInformationInteraction(provider)
     await getUsersInteraction(
       provider,
-      'search for a user that doesn\'t exists',
+      'check user existence with non-existing user',
```

Comment:
```suggestion
      'check user existence with non-existing users',
```

Reply:
`users` would be incorrect as it is more like checking user existence using one non-existing user in the request